Lib/test/test_types.py

```diff
@@ -604,6 +602,15 @@ def test_slot_wrapper_types(self):
         self.assertIsInstance(object.__lt__, types.WrapperDescriptorType)
         self.assertIsInstance(int.__lt__, types.WrapperDescriptorType)
 
+    # TODO: No signature found in builtin method __get__ of 'method_descriptor'
+    # objects so ValueError is raised
```

Comment:
Nit, we append `RUSTPYTHON` here to be able to differentiate if this is our `TODO` or not. 

```suggestion
    # TODO: RUSTPYTHON No signature found in builtin method __get__ of 'method_descriptor' objects.
```

Reply:
Thanks for reviewing this. I will fix that.